packages/cli/lib/errorHandlers/apiErrors.js

```diff
@@ -53,9 +56,9 @@ function logValidationErrors(error, context) {
  * @param {ApiErrorContext}          context
  */
 function logApiErrorInstance(error, context) {
-  if (error.isAxiosError) {
+  if (isHubSpotHttpError(error)) {
     if (overrideErrors(error)) return;
-    const errorWithContext = getAxiosErrorWithContext(error, context);
+    const errorWithContext = getHubSpotHttpErrorWithContext(error, context);
     logger.error(errorWithContext.message);
```

Comment:
Just noting b/c I ran into this during my testing, `getUserFriendlyHttpErrorMessage` returns the error message so there's no need to use `.message` here